README.md

```diff
@@ -21,6 +23,16 @@
 ~/.microsoft/usersecrets/ae59f009-796a-43af-9da1-816735206ea9/secrets.json
 ```
 
+### Install libman
+
+Libman installs client-side dependancies.
+
+```
```

Comment:
Handy trick: if you throw the name of the language immediately after the three backticks, it'll do syntax highlighting too!

```suggestion
```bash
```